### ISSUE TITLE
full symbol id does not render symbol in search pane

### ISSUE THREAD

**Issue by NatalieCampos** [Esri/military-symbology#180]
The documentation state that the full symbol id could be used or part of a symbol id. However when the full symbol id is entered no results are generated.  To reproduce:

1.  Enter a full symbol id (example: SUGPUSAPB).  Additional examples below:
![image](https://user-images.githubusercontent.com/5116835/27243427-b56e2cd8-5296-11e7-97b9-11678748cf09.png)
2. Press enter
3.  There will be no results.
![image](https://user-images.githubusercontent.com/5116835/27243520-2e47e996-5297-11e7-98ca-b05874419947.png)
4. if the last 4 digits are removed results will be returned.
![image](https://user-images.githubusercontent.com/5116835/27243481-f3b7da8e-5296-11e7-9240-4ef5125c0213.png)

**Comment by Dbarnes1** [comment]
@NatalieCampos this is similar to https://github.com/Esri/military-symbol-editor-addin-wpf/issues/178, I offered my input for a possible solution where the tags could be enhanced with the stylx. Hope this helps!

**Comment by topowright** [comment]
The most simple version of the symobol used above is this:  (SUGPUSAPBC-----)
![image](https://user-images.githubusercontent.com/5488313/27410601-72b470fe-569d-11e7-9643-dc4fe61c047b.png)

I have been able to draw this symbol using ArcGIS Pro so this proves that it has been entered in the key
![Uploading image.png…]()

**Comment by topowright** [comment]
This is not an issue that affects documentation. This issue has more to do with tags and keys in the sql lite database.  Another example would be: 

SFGPUCI---AF---
SFGPUCI---*****

This issue is also related to #182

**Comment by topowright** [comment]
The two main issues that I think are concerning is that we do not always honor the second letter to be F H N U. In order to make this symbol draw all the time you have to put in a * for the second symbol and then change the label to the proper identity.

**Comment by topowright** [comment]
After thinking about this issue over night I think this does affect documentation. The reason being is that not all symbols will allow a user to put in: 

SUGPUCI---*****

Instead for some symbols the user must put in: 

S*GPUCI---*****

I think identifying that the second symbol should be a * at times should be added to the documentation known issues.

**Comment by nfeuerstein** [comment]
Thanks @topowright - here's my thoughts: We don't expect the user to hand jam symbol id codes into the Military Symbol Editor. Those using the MSE will almost always be searching for a symbol using real human-readable words.  What we could expect is for the user to import a spreadsheet, a shapefile, or a service that does have symbol id codes. So, here's the biggest question for me: Is the issues we are seeing in the MSE and underlying/programatic issue with the lookup/stylx function. Does this same issue appear outside of the MSE?

**Comment by topowright** [comment]
To explain this issue that I think needs to be documented with an example I am going to use the symbol code that was used by @NatalieCampos in the beginning of this issue:  
SUGPUSAPB. 

She is correct that this symbol code does not work due to missing tags in the SQL lite database. If the symbol code is changed to : 
S*GPUSAPB 
The MSE will be able to find the symbol.  

This is not always the case when dealing with other symbols such as the symbol below
SHGPUCI---*****

![image](https://user-images.githubusercontent.com/5488313/27449274-a0966c76-573c-11e7-949f-c5a69d441a05.png)

**Comment by topowright** [comment]
@nfeuerstein THe issue we have found above does not happen outside the MSE. I think we can just support this issue by saying that the MSE does not fully support searching for a symbol using  an SIDC in the known issues section of the documentation.  I am just placing what we spoke about this morning here for clarification.

**Comment by topowright** [comment]
I have added this to the known issues list. I am now moving this issue to version after next. I have also removed all of the workflow tags.

**Comment by topowright** [comment]
This issue should have been closed back in June based on the fact that Military Symbol Editor was not designed to support SIDC codes being inputted by the user. This would be done by using the core dictionary render if necessary.

**Comment by topowright** [comment]
Based on new issues coming in from customers we will now need to support symbol id codes in the MSE

**Comment by csmoore** [comment]
I don't think this would be difficult to do with a simple workflow to just to have the symbol show up in the search (Option 1):

Option 1:

1. Do a regular expression match to see if the search term matches a SIDC
2. Strip out everything but the function (2525B/C) or entity code (2525D) and search on that and display the results

A more complex workflow (that the editor does not currently support) would be:

Option 2: 

If you wanted a symbol with **all** of the SIDC attributes set to then immediately open in the  symbol tab.

Yet another option (which is how MOLE used to work) is

Option 3: 

Add a text box somewhere (like the symbol tab) that shows the SIDC and allows you to manually enter and edit the SIDC
